client/packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/RequestStats.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Box } from '@openmsupply-client/common';
+import { DraftRequestLine } from '../hooks';
+import { StockDistribution } from './StockDistribution';
+import { ConsumptionHistory } from './ConsumptionHistory';
+import { StockEvolution } from './StockEvolution';
+
+export interface RequestStatsProps {
+  draft?: DraftRequestLine | null;
+}
+
+export const RequestStats = ({ draft }: RequestStatsProps) => {
+  return (
+    <Box
+      sx={{
+        minHeight: 200,
+        maxHeight: 500,
```

Comment:
maxing height to get allow for scrolling since graphs won't fit on smaller screens like this >.>

Reply:
Side by side is also wide.. not sure this is a design question